src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -94,8 +94,10 @@ void Graph::CreateGraph(const std::vector<NodePtr>& graphNodes,
     for (auto node : graphNodes) {
         if ("Parameter" == node->getTypeStr()) {
             inputNodesMap[node->getName()] = node;
```

Comment:
@ilya-lavrenov Any suggestion about this "inputNodesMap" which was used to match in
https://github.com/openvinotoolkit/openvino/blob/1be87f21b1967bc6dc625a0364d68367ecc0abee/src/plugins/intel_cpu/src/infer_request.cpp#L487
How to get the "FoundPort" info from the node?

Reply:
`inputNodesMap` should operate with indexes instead of names